app/lib/bk/compat/parsers/bitrise/steps.rb

```diff
@@ -5,7 +5,7 @@ module Compat
     module BitriseSteps
       # Implementation of Bitrise step translations
       class Translator
-        VALID_STEP_TYPES = %w[bundler brew-install change-workdir git-clone git-tag script].freeze
+        VALID_STEP_TYPES = %w[bundler brew-install change-workdir git-clone git-tag script github-status].freeze
```

Comment:
just nit-picking here, but it would be great to keep the alphabetical order here:

```suggestion
        VALID_STEP_TYPES = %w[bundler brew-install change-workdir git-clone github-status git-tag script].freeze
```